.prettierrc.js

```diff
@@ -2,6 +2,5 @@ module.exports = {
 	htmlWhitespaceSensitivity: 'ignore',
 	printWidth: 120,
 	singleQuote: true,
-	useTabs: true,
```

Comment:
This is enforced via `editorconfig`, see https://github.com/directus/directus/pull/18211/files#r1168558387. We shouldn't override it here in the `prettierrc.js`.

Reply:
Is this a statement or a request for change? Cause as far I as I can see, you made this change 😄